ksqldb-rest-app/src/main/java/io/confluent/ksql/rest/server/restore/KsqlRestoreCommandTopic.java

```diff
@@ -400,7 +400,7 @@ private static void maybeCleanUpQuery(final byte[] command, final KsqlConfig ksq
     boolean sharedRuntimeQuery = false;
     String queryId = "";
     final JSONObject jsonObject = new JSONObject(new String(command, StandardCharsets.UTF_8));
-    if (hasKey(jsonObject, "plan")) {
+    if (hasKey(jsonObject, "plan") && !jsonObject.isNull("plan")) {
```

Comment:
@vcrfxia @jnh5y -- build was failing due to the change in `PlanJsonMapper.java`: 

https://github.com/confluentinc/ksql/pull/8949/files#diff-df73b0e9550f5c1f929e6d5758135b5947bbbf44af70ac27bac08603091d0806L47-R49

The test indeed called System.exit that is why the Jenkins output was hard to read... The plan can now be `null` so we need to guard for this condition.